README.md

```diff
@@ -1,9 +1,23 @@
 # Summary
 
-An OMERO client for downloading data in bulk from the server. The `-h`
-option prints a brief summary of command-line options.
+An OMERO client for downloading data in bulk from an OMERO.server.
+The `-h` option prints a brief summary of command-line options.
 
-From the source repository `mvn` builds and packages the downloader.
+
+# Install
+
+Binaries can be downloaded from the `releases` page.
+
+NB: Use OMERO.downloader `0.1.5` to work with OMERO.server `5.4.x`.
+Use OMERO.downloader `0.2.x` to work with OMERO.server `5.5.x`.
+
+For development, from the source repository `mvn` builds and packages the downloader.
+
+
+# User Guide
+
+In addition to the instructions below, you can find a user-guide at
+https://omero-guides.readthedocs.io/en/latest/download/docs/download.html#image-export-using-omero-downloader-demo-only
```

Comment:
Outwith this PR, it might be nice to shorten the URL in the guide to prevent future breakgages cc: @jburel

Reply:
Could also provide explicit link text.